lib/elixir/lib/option_parser.ex

```diff
@@ -479,7 +479,7 @@ defmodule OptionParser do
       ["--bool", "--no-bool"]
```

Comment:
A few lines above, there is a backticked `options` that does not match the argument name `opts`. There is also a backticked `switches` that I guess should be `:switches` (because I think it’s talking about the option).

Reply:
fixed. That's why I am a big fan of never naming arguments `opts` but `options`